mpas_analysis/ocean/climatology_map_woa.py

```diff
@@ -122,6 +120,51 @@ def __init__(self, config, mpasClimatologyTask,
             comparisonGridNames=comparisonGridNames,
             iselValues=None)
 
+        remapAnnObsSubtask = None
+        remapMonObsSubtask = None
+
+        if controlConfig is None:
+            # Since we have a WOA18 annual climatology file and
+            # another file containing the 12 WOA18 monthly climatologies,
+            # do the remapping for each season separately
+
+            observationsDirectory = build_obs_path(
+                    config, 'ocean', 'woa18Subdirectory')
+
+            refFieldNames = [field['obsFieldName'] for field in fields]
+
+            if 'ANN' in seasons:
+                obsFileName = \
+                    '{}/woa18_decav_04_TS_ann_20190829.nc'.format(
+                        observationsDirectory)
+                remapAnnObsSubtask = RemapWoaClimatology(
+                    parentTask=self, seasons=['ANN'],
+                    fileName=obsFileName,
+                    outFilePrefix='woa18_ann',
+                    fieldNames=refFieldNames,
+                    depths=depths,
+                    comparisonGridNames=comparisonGridNames,
+                    subtaskName='remapObservationsAnn')
+                self.add_subtask(remapAnnObsSubtask)
+
+            seasonsMinusAnn = list(seasons)
+            if 'ANN' in seasonsMinusAnn:
+                seasonsMinusAnn.remove('ANN')
+            if len(seasonsMinusAnn) > 0:
+                obsFileName = \
+                    '{}/woa18_decav_04_TS_mon_20190829.nc'.format(
+                        observationsDirectory)
+                remapMonObsSubtask = RemapWoaClimatology(
+                        parentTask=self, seasons=seasonsMinusAnn,
+                        fileName=obsFileName,
+                        outFilePrefix='woa18_mon',
+                        fieldNames=refFieldNames,
+                        depths=depths,
+                        comparisonGridNames=comparisonGridNames,
+                        subtaskName='remapObservationsMon')
+
+                self.add_subtask(remapMonObsSubtask)
+
```

Comment:
We now create up to 2 obs. remapping tasks (one for `ann` and one for `mon`) rather than separate ones for T and S and for each season.